code/modules/client/preferences_gear.dm

```diff
@@ -518,6 +518,18 @@ var/global/list/gear_datums = list()
 	cost = 4
 	slot = WEAR_IN_BACK
 
+/datum/gear/weyland_booze
+	display_name = "Weyland-Yutani Lite"
+	path = /obj/item/reagent_container/food/drinks/cans/beer
+	cost = 1
+	slot = WEAR_IN_BACK
+
+/datum/gear/weyland_IPA
+	display_name = "Weyland-Yutani IPA"
+	path = /obj/item/reagent_container/food/drinks/cans/beer
+	cost = 1
```

Comment:
```suggestion
	cost = 2
	slot = WEAR_IN_BACK

/datum/gear/weyland_IPA
	display_name = "Weyland-Yutani IPA"
	path = /obj/item/reagent_container/food/drinks/cans/beer
	cost = 2
```
it should be at least SOMEWHAT hard for marines to get 7 cans of beer (that's enough for like half an entire squad)

Reply:
ok xeno main dev